source/faq.rst

```diff
@@ -216,5 +216,63 @@ You can change the minimum resolution for metrics by the following way:
    when |pmm-server| and |pmm-client| are on different networks,
    or when :ref:`pmm.amazon-rds`.
 
+.. _alertmanager:
+
+How to integrate Alertmanager with PMM?
+================================================================================
+
+The `Alertmanager <https://github.com/prometheus/alertmanager>`_ is the high
+performance solution developed by the Prometheus project to handle alerts sent
+by Prometheus. Handling includes deduplicating, grouping, and
+sending alert messages to the specified receiver via e-mail or other
+pre-configured channels. It also takes care of silencing and inhibition of
+alerts if needed.
```

Comment:
Feels like we're leading with what prometheus did and not what we did...I'd suggest flipping the first and second paragraphs so we first talk about what we've done to enable robust alerting from the PMM platform and only then tell people who don't already know what Alertmanager is.